.prod/on_deploy.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+echo "on_deploy.sh here ..."
+
+python /app/manage.py migrate --noinput
+
+# Generate the admin user using the password given in the environment variables.
+# If no password is set, the admin user gets a generated password which will
+# be written in stdout so that it can be accessed during the initial deployment.
+if [[ "$ADMIN_USER_PASSWORD" ]]; then
+    python /app/manage.py add_admin_user -u admin -p $ADMIN_USER_PASSWORD -e admin@example.com
+else
+    python /app/manage.py add_admin_user -u admin -e admin@example.com
+fi
+
+python /app/manage.py create_initial_page_content
```

Comment:
I wonder if this will cause problems in staging/production like persistent environments since the script is doing: 

`PageContent.objects.create` instead of e.g. `PageContent.objects.get_or_create`?

Reply:
Okay, my bad, I was checking things from `master` branch, which was not the right branch to check from. Everything seems to be in order, please carry on.

Reply:
![this-is-fine](https://user-images.githubusercontent.com/54528563/84655802-4c293180-af1a-11ea-844f-0fd7d30ded9d.PNG)